packages/bpk-component-navigation-bar/src/BpkNavigationBar.tsx

```diff
@@ -107,7 +111,10 @@ const BpkNavigationBar = (props: Props) => {
           className={getClassNames(
             'bpk-navigation-bar__trailing-item',
             `bpk-navigation-bar__trailing-item-${barStyle}`,
+            // if the Button has children safe to assume it's not an icon
+            !trailingButton.props?.children && 'bpk-navigation-bar__trailing-item--icon'
```

Comment:
The trailingButton prop isn't necessary an `Icon` it can be a `Link` and in that case we shouldn't set the icon style which adds the "Hug".

Reply:
Perhaps the prop should be renamed to trailingItem like the Styling is? For another PR perhaps.